app/packages/core/src/components/Actions/Options.tsx

```diff
@@ -281,7 +281,12 @@ const Options = ({ modal, anchorRef }: OptionsProps) => {
   const view = useRecoilValue(fos.view);
 
   return (
-    <Popout modal={modal} fixed anchorRef={anchorRef}>
+    <Popout
+      style={{ minWidth: "250px" }}
```

Comment:
Is there a reason the width can't be auto?

Reply:
Was being lazy. Changed it in `TabOption` in the next commit